torch/_inductor/post_grad_custom_pass.py

```diff
@@ -0,0 +1,53 @@
+import hashlib
+from abc import ABC, abstractmethod
+from functools import lru_cache
+from typing import Tuple, Union
+
+import torch.fx.graph
+
+
+class PostGradCustomPass(ABC):
```

Comment:
Nit: Can we make this more generic of a name? There are also options for joint graph, and pre grad? CompilePass maybe ? idk